guides/WixApps.md

```diff
@@ -1,20 +1,26 @@
-# About Wix Business Solutions
+# About Apps Made By Wix
+
+Wix sites can contain one or more apps made by Wix. Some API calls require a reference to the specific app made by Wix that it relates to. For example, when calling the [Create Comment](https://dev.wix.com/docs/rest/api-reference/comments/comments/create-comment) API, you must pass an `appId` to specify where to create the comment, for example, in Wix Blogs. Additionally, some API calls return a reference to the app made by Wix. For example, calling the [`Get a Coupon`](https://dev.wix.com/docs/rest/api-reference/coupons/coupons/get-a-coupon) API returns a coupon object with an `appId` referring to the ID of the Wix app that created the coupon, such as Wix Bookings or Wix Events. 
+
+See the following apps made by Wix and their app IDs:
+
+| **App made by Wix**    | **app ID**                           |
+|------------------------|--------------------------------------|
+| Wix eCommerce          | 1380b703-ce81-ff05-f115-39571d94dfcd |
+| Wix Stores             | 215238eb-22a5-4c36-9e7b-e7c08025e04e |
+| Wix Bookings           | 13d21c63-b5ec-5912-8397-c3a5ddb27a97 |
+| Wix Events             | 140603ad-af8d-84a5-2c80-a0f60cb47351 |
+| Wix Paid Plans         | 1522827f-c56c-a5c9-2ac9-00f9e6ae12d3 |
+| Wix Pay Button         | 3575d251-42c3-4992-adff-170b2af90a2c |
+| Wix Invoices           | 13ee94c1-b635-8505-3391-97919052c16f |
+| Wix Music              | 13bb5d67-1add-e770-a71f-001277e17c57 |
+| Wix Video              | 14409595-f076-4753-8303-9a86f9f71469 |
+| Wix Hotels             | 135aad86-9125-6074-7346-29dc6a3c9bcf |
+| Wix Blog               | 14bcded7-0066-7c35-14d7-466cb3f09103 |
+| Wix Subscriptions      | 8725b255-2aa2-4a53-b76d-7d3c363aaeea |
+| Wix Restaurants Orders | 13e8d036-5516-6104-b456-c8466db39542 |
```

Comment:
I find it hard to believe that Wix Restaurants Orders has its own app ID but not Wix Restaurants overall... what about Wix Restaurants Menus, then?